apps/publications/src/app/services/side-menu-items.service.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SideMenuItemsService {
+
+  constructor() { }
+
+  getSideMenuItems(): SideMenuItem[] {
+    const items: SideMenuItem[] = [];
+    items.push({
+      label: 'MENU_ITEMS.ALL_PUBLICATIONS',
+      icon: 'perun-publications-white',
+      isSVG: true,
+      link: '/all-publications',
+      activatedRegex: '^/all-publications$',
+      tabName: 'all-publications'
+    });
+    items.push({
+      label: 'MENU_ITEMS.MY_PUBLICATIONS',
+      icon: 'local_library',
+      link: '/my-publications',
+      activatedRegex: '^/my-publications$',
+      tabName: 'my-publications'
+    });
+    items.push({
+      label: 'MENU_ITEMS.CREATE_PUBLICATION',
+      icon: 'add_box',
+      link: '/create-publication',
+      activatedRegex: '^/create-publication$',
+      tabName: 'create-publication'
+    });
+    items.push({
+      label: 'MENU_ITEMS.AUTHORS',
+      icon: 'groups',
+      link: '/authors',
+      activatedRegex: '^/authors$',
+      tabName: 'authors'
+    });
+    items.push({
+      label: 'MENU_ITEMS.CATEGORIES',
+      icon: 'all_inbox',
+      link: '/categories',
+      activatedRegex: '^/categories$',
+      tabName: 'categories'
+    });
+    items.push({
+      label: 'MENU_ITEMS.PUBLICATION_SYSTEMS',
+      icon: 'assignment',
+      link: '/publication-systems',
+      activatedRegex: '^/publication-systems$',
+      tabName: 'publication-systems'
+    });
+    return items;
```

Comment:
Can you do checks here for authorizations please?

Reply:
No, because we don't have all the policies for methods that may be used here yet.